src/governance/governanceTransactionsFactory.spec.ts

```diff
@@ -0,0 +1,106 @@
+        assert.equal(transaction.data.toString(), `proposal@${Buffer.from(commitHash).toString("hex")}@0a@0f`);
+    });
+
+    it("should create transaction for voting", function () {
+        const transaction = factory.createTransactionForVoting(alice, {
+            proposalNonce: 1,
+            vote: Vote.YES,
+        });
+
+        assert.equal(transaction.sender.toBech32(), alice.toBech32());
+        assert.equal(transaction.receiver.toBech32(), governanceAddress);
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.chainID, config.chainID);
+        assert.equal(transaction.gasLimit, 5_171_000n);
+        assert.equal(transaction.data.toString(), "vote@01@796573");
+    });
+
+    it("should create transaction for closing proposal", function () {
+        const transaction = factory.createTransactionForClosingProposal(alice, {
+            proposalNonce: 1,
+        });
+
+        assert.equal(transaction.sender.toBech32(), alice.toBech32());
+        assert.equal(transaction.receiver.toBech32(), governanceAddress);
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.chainID, config.chainID);
+        assert.equal(transaction.gasLimit, 50_074_000n);
+        assert.equal(transaction.data.toString(), "closeProposal@01");
+    });
+
+    it("should create transaction for clearing ended proposals", function () {
+        const transaction = factory.createTransactionForClearingEndedProposals(alice, {
+            proposers: [alice, Address.newFromBech32("erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx")],
+        });
+
+        assert.equal(transaction.sender.toBech32(), alice.toBech32());
+        assert.equal(transaction.receiver.toBech32(), governanceAddress);
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.chainID, config.chainID);
+        assert.equal(transaction.gasLimit, 150_273_500n);
+        assert.equal(
+            transaction.data.toString(),
+            "clearEndedProposals@0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8",
+        );
+    });
+
+    it("should create transaction for claiming accumulated fees", function () {
+        const transaction = factory.createTransactionForClaimingAccumulatedFees(alice);
+
+        assert.equal(transaction.sender.toBech32(), alice.toBech32());
+        assert.equal(transaction.receiver.toBech32(), governanceAddress);
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.chainID, config.chainID);
+        assert.equal(transaction.gasLimit, 1_080_000n);
+        assert.equal(transaction.data.toString(), "claimAccumulatedFees");
+    });
+
+    it("should create transaction for changing config", function () {
+        const transaction = factory.createTransactionForChangingConfig(alice, {
+            proposalFee: "1000000000000000000000",
```

Comment:
also here you can create const for each

Reply:
don't think it's really necessary since they are used only once.